kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -249,7 +300,11 @@
     cursor: pointer;
 }
 
-
+.kb-data-staging__container {
```

Comment:
Added in some padding to the full data import container so that we could have more space